packages/data-stores/src/site/controls.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Internal dependencies
+ */
+import { wpcomRequest, WpcomClientCredentials } from '../utils';
+import { ActionType, CreateSiteAction } from './types';
+
+export default function createControls( clientCreds: WpcomClientCredentials ) {
+	return {
+		[ ActionType.CREATE_SITE ]: async ( action: CreateSiteAction ) => {
+			const { authToken, ...providedParams } = action.params;
+			const defaultParams = {
+				blog_title: '',
+				public: -1,
+				blog_name: '',
+				client_id: clientCreds.client_id,
+				client_secret: clientCreds.client_secret,
+				lang_id: 1,
+				locale: 'en',
+				options: {
+					theme: 'pub/maywood',
+					site_segment: 1,
+					site_vertical: 'p1',
+					site_vertical_name: 'Restaurant',
+					site_information: {
+						title: 'Test Site for Gutenboarding',
+					},
+					site_creation_flow: 'gutenboarding',
+				},
+			};
+			const mergedParams = Object.assign(
+				{},
+				defaultParams,
+				{ ...providedParams },
+				{ validate: false } // Set to false because account validation should be a separate action
+			);
+			const newUser = await wpcomRequest( {
```

Comment:
Not `newUser`, this is for sites 🙂 

If we're not processing the response here, we should be able to just return the promise. An async function is going to return a promise anyways, so unwrapping to return the result wrapped in a promise should be the same as `return wpcomRequest( /* … */ )`